roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -97,12 +97,39 @@ def get_var(self, *keys, **kwargs):
         OpenShiftCheckException when a key is not found or returning a default value
         provided as a keyword argument.
         """
+        if len(keys) == 1:
+            keys = keys[0].split(".")
+
         try:
             value = reduce(operator.getitem, keys, self.task_vars)
         except (KeyError, TypeError):
             if "default" in kwargs:
                 return kwargs["default"]
             raise OpenShiftCheckException("'{}' is undefined".format(".".join(map(str, keys))))
+
+        convert = kwargs.get("type")
```

Comment:
Suggestion:

```python
convert = kwargs.get("type", lambda x: x)
try:
    return convert(value)
except ValueError:
    raise OpenShiftCheckException(...)
```

Wherever we use `.get_var(...)`, we'd then always pass a function to the `type` optional argument, and never a string. Passing a string would be more cumbersome anyway.